main.go

```diff
@@ -869,12 +870,14 @@ is available in the $BITRISE_IDEDISTRIBUTION_LOGS_PATH environment variable`)
 
 	log.Donef("The xcarchive path is now available in the Environment Variable: %s (value: %s)", bitriseXCArchivePthEnvKey, tmpArchivePath)
 
-	if cfg.IsExportXcarchiveZip == "yes" {
-		if err := utils.ExportOutputDirAsZip(tmpArchivePath, archiveZipPath, bitriseXCArchiveZipPthEnvKey); err != nil {
-			fail("Failed to export %s, error: %s", bitriseXCArchiveZipPthEnvKey, err)
-		}
+	if err := utils.ExportOutputDirAsZip(tmpArchivePath, archiveZipPath, bitriseXCArchiveZipPthEnvKey); err != nil {
+		fail("Failed to export %s, error: %s", bitriseXCArchiveZipPthEnvKey, err)
+	}
+
+	log.Donef("The xcarchive zip path is now available in the Environment Variable: %s (value: %s)", bitriseXCArchiveZipPthEnvKey, archiveZipPath)
 
-		log.Donef("The xcarchive zip path is now available in the Environment Variable: %s (value: %s)", bitriseXCArchiveZipPthEnvKey, archiveZipPath)
+	if err := command.CopyFile(archiveZipPath, xcarchiveZipPath); err != nil {
```

Comment:
In case of the default input value for output dir, this will copy from the deploy dir to the deploy dir.